### ISSUE TITLE
Disabling esnext in file fails

### ISSUE THREAD

**Issue by revathskumar** [eslint/eslint#4637]
Disabling esnext in a file failes with error 

```js
/* eslint esnext: false */
```
Error Message 
```
Configuration for rule "esnext" is invalid: Value "false" is the wrong type
```

Version : v1.10.2 

![screenshot from 2015-12-08 16 06 27](https://cloud.githubusercontent.com/assets/463904/11662709/20c87e24-9dfe-11e5-9273-5c6a4f3bb53e.png)

At some point I received this message as well

```
Configuration for rule "esnext" is invalid:
Severity should be one of the following: 0 = off, 1 = warning, 2 = error (you passed "false").
```

**Comment by eslintbot** [comment]
@revathskumar Thanks for the issue! If you're reporting a bug, please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
2. What you did (the source code and ESLint configuration)
3. The actual ESLint output complete with numbers
4. What you expected to happen instead

Requesting a new rule? Please see [Proposing a New Rule](http://eslint.org/docs/developer-guide/contributing/new-rules) for instructions.

**Comment by hzoo** [comment]
I don't think esnext is a valid option in eslint (you are probably thinking of jshint)?

**Comment by revathskumar** [comment]
@hzoo I was using this with eslint version 1.7.2.

**Comment by platinumazure** [comment]
Inline configuration should use the same error level values as config files: 0, 1, or 2; and if any options are passed, they and the error level should be in an array.

For example:

```js
/* eslint semi: 0 */
/* eslint quotes: [2, "double"] */
```

In addition, as mentioned earlier, `esnext` does not correspond to an ESLint rule. (We don't have "options" for linting files really, but rather "rules" that can be enabled or disabled.)

Are you trying to convert from jshint/jslint to eslint? If so, we'll need more detail about what you're trying to do. A *rough* equivalent of "esnext" might be to turn on `es6` environment in your ESLint config file (or pass it as a CLI option).

**Comment by nzakas** [comment]
@revathskumar we added configuration validation since 1.7.2 You were using a configuration that wasn't doing anything previously, you just didn't know because there was no error.

**Comment by revathskumar** [comment]
ok. I enabled es6 in environment. Now how can I switch off es6 environment in a particular file?

**Comment by nzakas** [comment]
You can't turn off environments in files currently. That's because environments can change how the parser works.
